examples/nextjs-swc/src/locales/en.po

```diff
@@ -13,34 +13,35 @@ msgstr ""
 "Language-Team: \n"
 "Plural-Forms: \n"
 
-#: src/components/Switcher.tsx:14
+#: src/components/Developers.tsx:20
+msgid "{selected, plural, one {Developer} other {Developers}}"
+msgstr "{selected, plural, one {Developer} other {Developers}}"
+
+#: src/components/Switcher.tsx:10
 msgid "English"
 msgstr "English"
 
+#. js-lingui-explicit-id
+#: src/components/AboutText.tsx:6
+msgid "next-explanation"
```

Comment:
@yunsii for explicit custom ids `lingui@4` requires `#. js-lingui-explicit-id` comment to be added. This was a root cause why this section was not translated.